### PR TITLE
Add dag to check ingestion dag health

### PR DESCRIPTION
**Pull request by pankajastro**
closes: https://github.com/astronomer/ask-astro/issues/118

Sample alert

Screenshot 2023-11-14 at 1 26 35 PM

### CODE REVIEW

File: airflow/dags/monitor/monitor_ingestion_dags.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        logger.info("******************************")
+        message = ":red_circle: Import Error in DAG"
+
+    ingestion_dag_exist = False
+    if set(ingestion_dags).issubset(set(dagbag.dag_ids)):
+        ingestion_dag_exist = True
+        message = ":red_circle: Some Ingestion DAG's are missing"
+
+    if not ingestion_dag_exist or data:
+        print("hello hello")
```
Comment (sunank200) [inline airflow/dags/monitor/monitor_ingestion_dags.py:48]: We can remove this.
```suggestion
```

**Review by sunank200 — APPROVED** [review]
Have you tested this?

**Review by sunank200 — COMMENTED** [review]
@pankajastro have you tested this?

**Comment by pankajastro** [comment]
> @pankajastro have you tested this?

@sunank200 it is ready for review

**Comment by pankajastro** [comment]
> Have you tested this?

Yes @sunank200 I have just attached a sample alert message in PR description

**Pull request merged**